crates/file_finder/src/file_finder.rs

```diff
@@ -800,17 +800,18 @@ impl PickerDelegate for FileFinderDelegate {
             cx.notify();
             Task::ready(())
         } else {
-            let query = PathLikeWithPosition::parse_str(raw_query, |path_like_str| {
-                Ok::<_, std::convert::Infallible>(FileSearchQuery {
-                    raw_query: raw_query.to_owned(),
-                    file_query_end: if path_like_str == raw_query {
-                        None
-                    } else {
-                        Some(path_like_str.len())
-                    },
+            let query =
+                PathLikeWithPosition::parse_str(&raw_query, |formated_query, path_like_str| {
+                    Ok::<_, std::convert::Infallible>(FileSearchQuery {
+                        raw_query: formated_query.to_owned(),
```

Comment:
@mikayla-maki I have to change the `parse_str` callback with 2 arguments, because there need the replaced query, the `path_like_str` is not include the `line/col` suffix.